packages/ckeditor5-ui/tests/dropdown/utils.js

```diff
@@ -184,6 +184,62 @@ describe( 'utils', () => {
 				} );
 			} );
 
+			describe( 'closeDropdownOnBlur()', () => {
+				let externalFocusableElement, focusableDropdownChild;
+
+				beforeEach( () => {
+					externalFocusableElement = document.createElement( 'button' );
+					focusableDropdownChild = document.createElement( 'button' );
+
+					dropdownView.render();
+					dropdownView.panelView.element.appendChild( focusableDropdownChild );
+
+					document.body.appendChild( dropdownView.element );
+					document.body.appendChild( externalFocusableElement );
+				} );
+
+				afterEach( () => {
+					dropdownView.element.remove();
+					externalFocusableElement.remove();
+				} );
+
+				it( 'should close the dropdown when the focus was in the #panelView but it went somewhere else', done => {
```

Comment:
This and the next test are failing for me if browser running them is not focused. If they can't be rewritten to make them work in such a scenario, let's drop the comment explaining it.

Reply:
Addressed in bacb8cb.